### PR TITLE
Updates in sharing repos online

### PR DESCRIPTION
**Pull request by bast**
- new screenshots
- "main" branch is now the default in github web UI, I adapted the example but we may need to explain a bit

### CODE REVIEW

File: _episodes/09-remotes.md [hunk 1]
```diff
@@ -109,20 +107,35 @@ For the rest of this page, we will make a new repository for our
...
+<img src="{{ site.baseurl }}/img/creating-using-web/new-top-right.png" width="400px" style="border:2px solid #000000;">
+
+---
+
+Yet another way to create a new repository is to visit
+[https://github.com/new](https://github.com/new) directly.
+
+---
 
 On this page choose a project name (screenshot).
```
Comment (rkdarst) [inline _episodes/09-remotes.md:127]: Add screenshot, or point to screenshot below/above?  Or remove "(screenshot)"?

File: _episodes/09-remotes.md [hunk 2]
```diff
@@ -141,25 +154,25 @@ We now want to try the second option that GitHub suggests:
...
 2. Check that you are in the right place with `git status`.
-3. Copy paste the two lines to the terminal and execute those, in my case (**you
+3. Copy paste the three lines to the terminal and execute those, in my case (**you
   need to replace the "user" part and possibly also the repository name**):
 
 ```shell
-$ git remote add origin https://github.com/user/recipe.git
-$ git push -u origin master
+$ git remote add origin https://github.com/user/recipe.git
+$ git branch -M main
```
Comment (rkdarst) [inline _episodes/09-remotes.md:162]: This, by default, renames the current branch without an `<oldbranch>` argument?  The manual page doesn't specify, and it's a news to me you don't need that!

Can we say what these three lines mean?

**Review by rkdarst — APPROVED** [review]
Several comments below.  I haven't looked at the screenshots but I presume they are good.

File: _episodes/09-remotes.md (hunk 2 above)
Comment (bast) [reply _episodes/09-remotes.md:162]: I didn't know this flag until today.

**Pull request merged**